src/scopeHandler.js

```diff
@@ -0,0 +1,13 @@
+const qs = require('querystring')
+
+const scopes = {
+  decode: function (scopesString) {
+    if (Array.isArray(scopesString)) {
+      return scopesString
+    }
+    const decodedScopes = qs.parse(`scope=${scopesString}`)
```

Comment:
This handles:
`x+y+z`
`x y z`
`x%20y%20z`

Which are all the circumstances in urls we need